docs/ci-setup.md

```diff
@@ -0,0 +1,52 @@
+# Notes to Setup SGX VM for CI Testing
+
+# SGX PSW
+
+- install `aesmd`, configure with `default quoting type = ecdsa_256` and the preferred DCAP quote provider lib (with its config)
+
+# containerd
+We pre-install the CoCo containerd and tools with a simple command:
+
+```bash
+curl -fsSL https://github.com/confidential-containers/containerd/releases/download/v1.6.8.1/cri-containerd-cni-1.6.8.1-linux-amd64.tar.gz | sudo tar zx -C /
+```
+
+And it's configured with the following runtimehandler:
+```
+$ tail -3 /etc/containerd/config.toml
+[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.enclavecc]
+  cri_handler = "cc"
+  runtime_type = "io.containerd.rune.v2"
+```
+
+# verdictd (for EAA-KBC testing)
+- install [verdictd](https://github.com/inclavare-containers/verdictd) and its dependencies
+- [RATS-TLS](https://github.com/inclavare-containers/rats-tls) version is recommended to be the same as what the `agent-enclave` build uses.
+- `/usr/local/bin/verdictd --listen 127.0.0.1:1234 --verifier sgx_ecdsa --attester nullattester --client-api 127.0.0.1:12340 --mutual`
```

Comment:
verdictd used for https://github.com/confidential-containers/enclave-cc/pull/140
```bash
docker run  \
    -v $(pwd)/data:/opt/verdictd \
    -v /etc/sgx_default_qcnl.conf:/etc/sgx_default_qcnl.conf \
    -v /usr/local/lib/rats-tls:/usr/local/lib/rats-tls \
    --device /dev/sgx_enclave \
    --device /dev/sgx_provision \
    -p 12345:12345 \
    --env RUST_LOG=debug \
    -d xynnn007/verdictd:cc0.5.0-rc0 \
    verdictd --listen 0.0.0.0:12345 --client-api 127.0.0.1:50000 --verifier sgx_ecdsa --attester nullattester --mutual 
```

which relys on the `rats-tls` on the host.

Reply:
OK..see the comment about `ACTIONS_RUNNER_HOOK_JOB_STARTED` below. I think that conflicts with this.

Reply:
Oh yes. Actually there is a verdictd daemon always running on the CI machine, thus all tests set the `"key_provider"` to `"provider:attestation-agent:eaa_kbc::127.0.0.1:1234"`.

Now I started a new verdictd and listened to port `12345`.

The start-up and shut-down of the verdictd is not in the previous CI, neither the new CI